HACKING.txt

```diff
@@ -57,52 +57,28 @@ Coding Style
 Running Tests
 -------------
 
-- The `tox.ini` uses `nose` and `coverage`. As such `tox` may be used
+- The `tox.ini` uses `pytest` and `coverage`. As such `tox` may be used
   to run groups of tests or only a specific version of Python. For example, the
-  following command will run tests on Python 3.7 only without coverage:
+  following command will run tests on Python 3.8 only with coverage:
 
-    $ tox -e py37
+    $ tox -e py38
 
-  This command will run tests on the latest version of Python 3 with coverage.
-
-    $ tox -e py36-cover,coverage
-
-- To run individual tests (i.e., during development), you can use `nosetests`
+- To run individual tests (i.e., during development), you can use `pytest`
   syntax as follows, where `$VENV` is an environment variable set to the path
   to your virtual environment:
 
     # run a single test
-    $ $VENV/bin/nosetests tests.test_module:ClassName.test_mytestname
+    $ tox -e py38 -- --no-cov tests/test_httpexceptions.py::TestHTTPMethodNotAllowed::test_it_with_default_body_tmpl
```

Comment:
I prefer the abstract to the concrete.

If we drop `$VENV`, and to ease on-boarding, we should:
- remove `$VENV` from the previous sentence, and
- suggest a method for non-experts to install tox

```suggestion
    $ tox -e py38 -- --no-cov tests/test_module.py::ClassName::test_my_test_name
```

Reply:
@mmerickel what do you suggest for installing tox? I install it into the project's virtual environment, not that it's the best way, but it is easy to manage for on-boarding.

Reply:
@mmerickel suggested linking to https://tox.readthedocs.io/en/latest/install.html

I honestly don't think we should be in the business of telling people how to install their tools. In the case that we do have a $VENV it quickly becomes dangerous because if they install pyramid into that venv and run tests, then that venv may leak through tox and cause false positives or errors as things are being imported from the wrong location. This is even worse if the venv is activated by the user.

I hate to say it, but when was the last time we on-boarded anyone?

Reply:
A link is fine.

Search pull requests and issues for `HACKING.txt` as an indicator of on-boarding people who were referred to it when they attempted to make a PR and hit issues. I know I do it several times a year.

Reply:
I've updated to add installation of `tox` using `pip3 install --user`.